src/components/Routes.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import{ Router, Switch, Route} from 'react-router'
+import Home from '../pages/Home'
+import Login from '../pages/Login'
+import NotFound from '../components/NotFound'
+
+import {history} from '../history'
+
+const Routes = () => (
+    <Router history = {history}>
+        <Switch>
+            <Route component={Login} exact path ="/login"/>
+            <Route component={Home} exact path ="/home"/>
+            <Route component={NotFound} exact path ="/"/>
```

Comment:
recomendável redirecionar a página inicial para *Home*, e as demais URl's que não existem para *NotFound*. Possível solução em: https://stackoverflow.com/questions/34735580/how-to-do-a-redirect-to-another-route-with-react-router